python/ray/serve/tests/unit/test_deployment_scheduler.py

```diff
@@ -227,7 +227,9 @@ def test_get_available_resources_per_node():
     d_id = DeploymentID("a", "b")
```

Comment:
All newly added tests in this PR already currently pass, I'm only adding them for better coverage. This modified test `test_get_available_resources_per_node` is the only one that fails without the change in this PR.